packages/bpk-svgs/tasks/metadata/automirror.js

```diff
@@ -0,0 +1,17 @@
+const fs = require('fs');
```

Comment:
Do we need to add the licence header here?

Reply:
yep, good catch